modules/background-task/ios/ReactNativeBackgroundTask.mm

```diff
@@ -0,0 +1,109 @@
+#import "ReactNativeBackgroundTask.h"
+#import <UIKit/UIKit.h>
+#import <BackgroundTasks/BackgroundTasks.h>
+#import "RNBackgroundTaskManager.h"
+
+@implementation ReactNativeBackgroundTask {
+    NSMutableDictionary *_taskExecutors;
+}
+
+RCT_EXPORT_MODULE()
+
+- (instancetype)init {
+    if (self = [super init]) {
+        _taskExecutors = [NSMutableDictionary new];
```

Comment:
Do you think we need any kind of concurrency safety for `_taskExecutors`?

Reply:
I don't think concurrency protection needed for `_taskExecutors` because:
- `defineTask` (`RCT_EXPORT_METHOD`) runs on main thread
- `BGTask` handlers run on main thread

These are the only two places accessing `_taskExecutors`. Adding synchronization would just add overhead with no benefit since everything already runs on the main thread.